packages/view-types/obs-sets-manager/src/ObsSetsManagerSubscriber.js

```diff
@@ -82,22 +81,22 @@ export function ObsSetsManagerSubscriber(props) {
     dataset,
     obsType,
     obsSetSelection: cellSetSelection,
+    obsSetExpansion: cellSetExpansion,
     obsSetColor: cellSetColor,
     additionalObsSets: additionalCellSets,
     obsColorEncoding: cellColorEncoding,
   }, {
     setObsSetSelection: setCellSetSelection,
     setObsColorEncoding: setCellColorEncoding,
     setObsSetColor: setCellSetColor,
+    setObsSetExpansion: setCellSetExpansion,
     setAdditionalObsSets: setAdditionalCellSets,
   }] = useCoordination(COMPONENT_COORDINATION_TYPES[ViewType.OBS_SETS], coordinationScopes);
 
   const title = titleOverride || `${capitalize(obsType)} Sets`;
 
   const [urls, addUrl] = useUrls(loaders, dataset);
 
-  const [cellSetExpansion, setCellSetExpansion] = useState([]);
-
```

Comment:
Can the change make `cellSetExpansion` a coordination type be reverted? - I think we discussed this as an option early on but it seems like now it can stay as local state

Reply:
I can't, because the logic in the current implementation relies on this information ([see filterPaths function](https://github.com/vitessce/vitessce/pull/1503/files#diff-6b2ae5efdd37c0a36a5039622c21787d71e07723c1037b9b69a805fe6a74b9bdR86))

Reply:
Ah my bad - fine to keep then!